tests/common/platform/interface_utils.py

```diff
@@ -44,6 +44,9 @@ def parse_intf_status(lines):
 
 
 def check_interface_status_of_up_ports(duthost):
+    if duthost.facts['asic_type'] == 'vs' and duthost.is_supervisor_node():
+        return True
+
     if duthost.is_multi_asic:
         up_ports = []
         for asic in duthost.frontend_asics:
```

Comment:
@BYGX-wcr for supervisor, do we need an explicit check here? Isnt this is_multi_asic case covering this scenario??(the asics on the sup will not be frontend asics??)

Can you explain more what is failing on VS?

Reply:
actually nvm.. this will be needed due to our new implementation of vs support for T2-Chassis.